modules/installation-gcp-shared-vpc-permissions.adoc

```diff
@@ -0,0 +1,21 @@
+// This file is referenced in the following assembly:
+// installing/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-permissions_{context}"]
+= Required GCP permissions for shared VPC installation
+
+When you attach the `Owner` role to the service account that you create, you
+grant that service account all permissions, including those that are required to
+install {product-title}. To deploy an {product-title} cluster into a shared VPC, the
+service account requires the following permissions in the host project.
+
+.Required roles for the installation program
+* Compute Admin
+* Compute Load Balancer Services User
+
+.Required roles for creating network resources during installation
+* Compute Network User
+* DNS Administrator
+
```

Comment:
```suggestion
* Compute Firewall Create
[NOTE]
====
If you do not want the installation program to automatically create firewall rules for your cluster, you do not need the Compute Firewall Create permission.
====
```
I think this would be a good place to add this information instead of the other XPN PR https://github.com/openshift/openshift-docs/pull/54736.  what do you think? is my wording accurate and is there a better name for the permission than "Compute Firewall Create"?